### PR TITLE
Document Enabling custom metrics API using kops

### PR DESCRIPTION
**Pull request by RahulMahale**
- Tested it with kops version 1.7 and k8s v1.7.2

### CODE REVIEW

File: docs/cluster_spec.md [hunk 1]
```diff
@@ -194,6 +194,16 @@ NOTE: Where the corresponding configuration value can be empty, fields can be se
 
 Will result in the flag `--resolv-conf=` being built.
 
+#### Enable Custom metrics support
+To use custom metrics in kubernetes we have to set flag  `ENABLE_CUSTOM_METRICS` to `true` on all the kubelets.
+we can specify that in the `kubelet` spec in our cluster.yml.
```
Comment (mikesplain) [inline docs/cluster_spec.md]: Maybe we should add a link to the custom metrics docs on kubernetes.io? 

https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-custom-metrics

Comment (RahulMahale) [reply docs/cluster_spec.md]: @mikesplain Yes, Have included that link. Thanks.

**Comment by RahulMahale** [comment]
@k8s-merge-robot rebased with master.

**Pull request merged**